Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -238,12 +251,17 @@ def create_quam_superconducting(
         },
     }
 
+    # TODO: let's first do everything with manual wiring for simplicity
     # port_allocation = default_port_allocation(
     #     num_qubits=2,
     #     using_opx_1000=using_opx_1000,
     #     starting_fem=3
     # )
+    # TODO: why not having this function part of create_wiring, since the user doesn't care about it?
     port_allocation = custom_port_allocation(custom_port_wiring)
+    # TODO: can we re-wire after the QuAM has been created?
+    #  Use-case: I have 5 qubits on my chip but only 1 OPX+Octave so I want to be able to update the wiring based on
+    #  the active qubits.
```

Comment:
Maybe we should separate this into two steps, something like:
```
if wiring_file.exists()
    wiring = json.load(wiring_file)
else:
    wiring = create_wiring(port_allocation, ...)
   json.dump(wiring, wiring_file)

machine = create_quam_superconducting(wiring)
```
This way we would have a JSON file that we could edit after creating QuAM. Wdyt?